lib/active_remote/attribute_methods.rb

```diff
@@ -9,11 +9,11 @@ def attribute_names
     end
 
     def [](name)
-      attribute(name)
+      attribute(name.to_s)
     end
 
     def []=(name, value)
-      write_attribute(name, value)
+      _write_attribute(name.to_s, value)
```

Comment:
This could just be changed to

```ruby
@attributes.write_from_user(name, value)
```

Reply:
`write_from_user` is not defined in this gem, is it? Pretty sure that is in `active_remote-set-by` which is an MX internal gem.

Reply:
Also, I am not sure where `_write_attribute` is defined and not sure it should be called from `active_remote` if it isn't already in a dependency, besides the fact that it is private.